### DISCUSSION TITLE
WhatsApp Messagebird Reply out going by the wrong channel.

### DISCUSSION THREAD

**Discussion by eloimarquessilva** [LiveHelperChat/livehelperchat#1973] [General]
Hello, I'm using two WhatsApp numbers integrated with Messagebird (https://doc.livehelperchat.com/docs/integrating/messagebird-whatsapp/). 

WhatsApp #1 is set up for the sales department. 
WhatsApp #2 is set up for the support department. 

If the customer is talking to both departments at the same time (or within a short period of time), the operator's response will always be sent to the customer via the channel (WhatsApp) that initiated the first conversation. 

For example, if the first conversation was initiated in the Support department (WhatsApp #2) and the customer initiates another conversation with the Sales department (WhatsApp #1), the responses from the sales department which should be sent via WhatsApp #1 will be sent through the channel WhatsApp #2 of the support department, not by WhatsApp #1.

This only affects sending the conversation in the direction "operator  -> customer". The conversation in the direction "customer -> operator" works normally through the correct channel.

Any idea how to fix this?
Thank you!

**Comment by remdex** [comment]
I think you are using older extnsion or older webhook configuration. If you have only one phone number for whatsapp, it's just not possible to know to what chat to assign messages. If two business account all has to work.

Make sure you incoming webhook configuration this section looks like this.
![image](https://github.com/LiveHelperChat/livehelperchat/assets/1146085/70f03f03-6ee7-444c-a088-a75abd92cfd7)

**Reply by eloimarquessilva** [reply]
First of all, thank you very much for the reply.

Sorry, I don't know if I understood well what you meant.
I'm not using any specific extension. The webhook setup was done on 2022-07-27.

I have three incoming webhooks. One for each WhatsApp and one for Facebook/Instagram.
![image](https://github.com/LiveHelperChat/livehelperchat/assets/11034410/c0a4215c-0e39-4274-b22d-e47b79526440)

The "main attributes" off all incomming webhooks are
![image](https://github.com/LiveHelperChat/livehelperchat/assets/11034410/0415421d-2427-4104-ad3a-ca8ede4527f4)

I can't location "Chat ID 2 field location". Are there any updates that need to be made to Live Helper Chat that could be causing this?

Thank you!

**Comment by remdex** [comment]
I don't know what kind of integration you have there. As all official integration has one webhook based on provider.... messagebird and fbmessenver and non of them have this problem...
